packages/eslint-plugin/src/rules/indent.ts

```diff
@@ -175,6 +175,15 @@ export default util.createRule<Options, MessageIds>({
         }
       },
 
+      VariableDeclaration(node: TSESTree.VariableDeclaration) {
+        // https://github.com/typescript-eslint/typescript-eslint/issues/441
+        if (node.declarations.length === 0) {
```

Comment:
👏👏👏
I faced the exact same issue and debugging the code I was right here to add this!  Thanks for being on top of it. <3 @bradzacher